lms/djangoapps/certificates/api.py

```diff
@@ -203,8 +204,43 @@ def get_recently_modified_certificates(course_keys=None, start_date=None, end_da
     if user_ids:
         cert_filter_args['user__id__in'] = user_ids
 
-    return GeneratedCertificate.objects.filter(**cert_filter_args).order_by('modified_date')
+    # Include certificates with a CertificateDateOverride modified within the
+    # given time range.
+    if start_date or end_date:
+        certs_with_modified_overrides = get_certs_with_modified_overrides(course_keys, start_date, end_date, user_ids)
 
+    return GeneratedCertificate.objects.filter(**cert_filter_args).union(certs_with_modified_overrides).order_by('modified_date')
+
+def get_certs_with_modified_overrides(course_keys=None, start_date=None, end_date=None, user_ids=None):
```

Comment:
Could we just use the modified date on the cert override table and achieve the same results?